cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -88,14 +89,19 @@ def test_duplicate_name_error_validation():
 @pytest.mark.xfail(message='http://cfme-tests.readthedocs.org/guides/gotchas.html#'
     'selenium-is-not-clicking-on-the-element-it-says-it-is')
 def test_name_max_character_validation():
-    """Test to validate name with maximum characters in customization templates."""
+    """Test to validate name with maximum characters in customization templates.
+       Max length is controlled by UI elements - we are not allowed to input more than we should
+       Opens template details to verify that extra symbols were cut
+    """
     template_name = pxe.CustomizationTemplate(
         name=fauxfactory.gen_alphanumeric(256),
         description=fauxfactory.gen_alphanumeric(16),
         image_type='RHEL-6',
         script_type='Kickstart',
         script_data='Testing the script')
-
-    with error.expected('Name is required'):
```

Comment:
logic here was changed - we are able to input only 255 symbols, everything else gets cutted => open details of template by slice and check it's lengths